File: File_Operations/read.py
from sys import argv

def readFile(fileName):
	file = open(fileName, 'r')
	for line in file:
		print(line.strip())
	file.close()

'''
	Two possible ways of reading from file. Either txt file is provided via command line argument 
	or file name is provided in function call.
	
	1) readFile(argv[1]) 
	> python3 read.py read.txt

	prints:
		This
		is
		a
		test
		file

	2) readFile('read.txt')
	> python3 read.py

	prints:
		This
		is
		a
		test
		file
'''

File: File_Operations/Read/read.js
var fs = require('fs')
var fileName = process.argv[2]

if(process.argv.length < 3){
	console.log('Provide a file path');
	process.exit(1);
}

fs.readFile(fileName, 'utf-8', (error, data) => {
	if(error){
		throw error;
	} else {
		console.log(data);
	}
})

/***

	node read.js read.txt

	prints:
		This
		is
		a
		test
		file
		
***/